documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,254 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-verify.adoc
+// getting-started.adoc
+
+[id='setup-external-clients-{context}']
+= Setting up access for clients outside of Kubernetes
+
+This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes.
+
+Using the address of the Kafka cluster, you can provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
+
+You configure an external Kafka listener to provide the access.
+
+The following external listener types are supported:
+
+* `route` to use OpenShift `Route` and the default HAProxy router
+* `loadbalancer` to use loadbalancer services
+* `nodeport` to use ports on Kubernetes nodes
+* `ingress` to use Kubernetes _Ingress_ and the {NginxIngressController}
+
+The type chosen depends on your requirements, and your environment and infrastructure.
+For example, `nodeport` is the least secure, so you might not wish to use it in a production environment.
```

Comment:
I would remove this sentence. It suggests it is insecure which is not trhough. Also the evaluation might be subjective. If you want some examples why you might use one or another:
* Loadbalancers might not be supported on all infras (e.g. bare metal), node ports might work fine there
* Loadbalancers often generate additional costs

etc.

Reply:
I dropped the sentence and replaced with:

> The type chosen depends on your requirements, and your environment and infrastructure.
> For example, loadbalancers might not be suitable for certain infrastructure, such as bare metal, where node ports provide a better option.